src/Illuminate/Database/Eloquent/Collection.php

```diff
@@ -132,6 +132,16 @@ public function modelKeys()
 	}
 
 	/**
+	 * Touch the models in the collection.
+	 *
+	 * @return void
+	 */
+	public function touchOwners()
+	{
+		$this->transform(function($m) { $m->touch(); });
```

Comment:
Please try to avoid singe line statements like this. They're fine for the tests, but not for src.